src/cmd/check.go

```
@@ -26,6 +28,13 @@ func Check(cfg *MainConfig) (int, error) {
 	if args.disableCache {
 		linter.CacheDir = ""
 	}
+	if args.gitRepo != "" {
+		gitDir, err := filepath.Abs(args.gitRepo + "/../")
+		if err != nil {
+			log.Fatalf("Find git dir: %v", err)
```

Don't do log fatal. You can return an error here.